examples/check.py

```diff
@@ -4,21 +4,21 @@
 sys.path.insert(0, os.path.abspath(__file__+'../../..'))
 
 # Replace this API key with your own.
-lob.api_key = "test_fc26575412e92e22a926bc96c857f375f8b"
+lob.api_key = "live_6289b7c522ea3567a5d7502cdf8129eb6e5"
```

Comment:
Hi there,

Looks like you left your live API key in –– please update this (and reset your live API key), thanks!

Reply:
I just copied it to file and edit completion of the information space, was the check mail out as i have done it

Reply:
how will i do this i thought i have copied it to the file and i want to know the check was mailed out